packages/service-clients/end-to-end-tests/odsp-client/package.json

```diff
@@ -88,6 +91,7 @@
 		"c8": "^8.0.1",
 		"eslint": "~8.55.0",
 		"nock": "^13.3.3",
+		"prettier": "~3.0.3",
```

Comment:
huh - prettier added to devDeps here, but I don't see this reflected in the lockfile.?

Reply:
Looks like prettier and also cross-env are in `dependencies` but should just be in `devDependencies`. Not sure about others.

Reply:
I can't explain why the lockfile doesn't seem to reflect this. I've gone through the "regen lockfile" dance a couple of times now as I have merged latest main and I get the same results each time.

Reply:
Because you need to remove prettier from the `"dependencies"` sections. I think cross-env should move to devDeps too.

Reply:
Ahh, I didn't see that because I didn't touch deps and there were no changes there. I'll clean it up.

Reply:
Fixed. I found a few other projects with the same problem so I opened #20171.